### PR TITLE
Init branch andrew

### PR DESCRIPTION
**Pull request by awoodcoc**
Hi! I've structured out the initial files for our project--Spring Boot initialized back end with a Reach Front end. Requesting a pull to main so that we can collaborate from this project from main.

### CODE REVIEW

File: root-app-frontend/public/logo192.png
Comment (jaylingelbach) [inline root-app-frontend/public/logo192.png:1]: Please remove all stock images

File: root-app-frontend/src/App.css
Comment (jaylingelbach) [inline root-app-frontend/src/App.css:1]: Please remove stock css

File: root-app-frontend/src/App.js
Comment (jaylingelbach) [inline root-app-frontend/src/App.js:1]: Remove stock html/jsx

File: root-app-frontend/src/App.test.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```
Comment (jaylingelbach) [inline root-app-frontend/src/App.test.js:6]: Remove stock html/jsx

File: root-app-frontend/src/index.css
Comment (jaylingelbach) [inline root-app-frontend/src/index.css:1]: Remove stock html/jsx

File: root-app-frontend/src/logo.svg
Comment (jaylingelbach) [inline root-app-frontend/src/logo.svg:1]: Remove stock images

**Review by jaylingelbach — CHANGES_REQUESTED** [review]
Good stuff, just remove stock code/images

**Comment by awoodcoc** [comment]
> Good stuff, just remove stock code/images

Thank you! All comments should be addressed, with stock JS/HTML/Images removed. Let me know if this looks good to merge.

**Pull request merged**